backend/src/main/java/clf/integra/backend/service/UserService.java

```diff
@@ -0,0 +1,12 @@
+package clf.integra.backend.service;
+import clf.integra.backend.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Fyi, however you prefer - If you use lombok, you can also simply use `@RequiredArgsConstructor` for injecting dependencies, no need for a manual constructor